lib/susedistribution.pm

```diff
@@ -392,6 +392,7 @@ sub init_consoles {
         $self->add_console('log-console',    'tty-console', {tty => 5});
         $self->add_console('displaymanager', 'tty-console', {tty => 7});
         $self->add_console('x11',            'tty-console', {tty => get_x11_console_tty});
+        $self->add_console('extra-console',  'tty-console', {tty => 3});
```

Comment:
I would recommend to make the name explicit, e.g. just 'check_keyboard_layout'

Reply:
I actually think makes more sense to have it named extra console instead :) check_keyboard_layout sounds too much like: "We need just this console, just for this.." which for the time being is true. But it's more an extra console in reality...

Reply:
@okurz Are you ok to stick to 'extra' for the name?

Reply:
well, alright